xbmc/video/VideoDatabase.cpp

```diff
@@ -5253,11 +5259,38 @@ void CVideoDatabase::UpdateTables(int iVersion)
     m_pDS->exec("DROP TABLE settings");
     m_pDS->exec("ALTER TABLE settingsnew RENAME TO settings");
   }
+  if (iVersion < 110)
+  {
+    m_pDS->exec("CREATE TABLE history(id INTEGER PRIMARY KEY, dateWatched TEXT NOT NULL, idFile INTEGER NOT NULL)");
+
+    m_pDS->exec("INSERT INTO history(dateWatched, idFile) SELECT lastPlayed, idFile FROM files AS f WHERE f.playCount is not NULL");
+
+    m_pDS->query("SELECT idFile, playCount FROM files WHERE playCount > 1");
+    // don't run for playcount == 1 as we already migrated those
+    while (!m_pDS->eof())
+    {
+      int idFile = m_pDS->fv(0).get_asInt();
+      int playCount = m_pDS->fv(1).get_asInt();
+      // as we have no dates for these, we'll add them with an empty string - shouldn't be the case otherwise - ever!
+      // add the item as often as we have a playcount, we've added the first item above, so we're skipping that one now
+      for (int i = 1; i < playCount; i++)
+      {
+        m_pDS->exec(PrepareSQL("INSERT INTO history(dateWatched, idFile) SELECT '', %i;", idFile));
```

Comment:
Why do you use insert into select and not insert into values?

Reply:
This was a real select before. But I don't think it's different in performance or readability?
Anyway can change it.